interface/themes/style_light.css

```diff
@@ -1505,3 +1499,9 @@ table.dataTable#therapy_groups_list tbody td{
 .sidebar {
   background-color: #f8f8f8;
 }
+
+.alert-danger {
+  font-size: 100% !important;
+  background-color: #FF0000 !important;
+  color: #fff !important;
+}
```

Comment:
Nope, this is a bootstrap class, I just overrode it in light because I wanted to give it a different look